design.md

```diff
@@ -2,12 +2,24 @@
 
 ## Who is the target for this design, e.g., are you assuming any knowledge on the part of the language users?
 
+Lord of the Rings fans who are moderately well-versed in the lore of the world.
+
```

Comment:
Interestingly, when we actually looked at the map to get representatives for North, South, East, and West, we realized that the representatives we ultimately chose were nowhere near to actual corner points for Middle Earth. However, we chose to keep them because each of those 4 locations generally had a canonical cardinal direction associated with them. It may have been more correct to have North be "toward Angmar", but I feel like more LOTR fans recognize the Shire than Angmar as a location ( though there's an argument to be made that those people aren't true fans... :stuck_out_tongue: )

![middleearthlargelargerstill](https://cloud.githubusercontent.com/assets/7916986/10326138/82e1ce32-6c4c-11e5-81d0-4dc2b045d6cf.jpg)